docs/serverless/08-05-create-git-function.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Create a Function from Git repository sources
+type: Tutorials
+---
+
+This tutorial shows how you can create a Function in an alternative way -  building it from code and dependencies stored in a Git repository instead of keeping the code in the Function CR. The tutorial is based on the Function from the [`orders service` example](https://github.com/kyma-project/examples/tree/master/orders-service). It describes steps required to fetch Function's source code and dependencies from a public Git repository that does not require any authentication method. However, it also provides additional guidance on how to secure it if you are using a private repository.
+
+> **NOTE:** This tutorial shows an alternative way of storing Function's code and dependencies. If you want to follow the whole end-to-end flow described in the Serverless tutorials, [create an inline Function](#tutorials-create-an-inline-function) instead. To learn more about Git repository sources for Functions and different ways of securing your repository, read about the [Git source type](#details-git-source-type).
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Export these variables:
+
+    ```bash
+    export GIT_FUNCTION={GIT_FUNCTION_NAME}
+    export NAMESPACE={FUNCTION_NAMESPACE}
+    ```
+
+2. Create a Secret (optional).
+
+    If you use a secured repository, you must first create a Secret for one of these authentication methods:
+
+    - Basic authentication (username and password) to this repository in the same Namespace as the Function:
```

Comment:
Maybe it would be worth mentioning that we also support using Github tokens. (to use token you have to use a `basic` secret, fill the username and paste token into `password` field)

Reply:
Good point :)